src/Nethermind/Nethermind.Evm/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -380,10 +380,10 @@ protected virtual TransactionResult ValidateStatic(
 
         protected virtual TransactionResult ValidateGas(Transaction tx, BlockHeader header, long minGasRequired, bool validate)
         {
-            if (tx.GasLimit < minGasRequired)
+            if (validate && tx.GasLimit < minGasRequired)
```

Comment:
This flag is confusing me, why should it exist here in the first place?
Is it something used in arbitrum? @damian-orzechowski ?
If not why call this method if it is false at all?